packages/devextreme/js/__internal/grids/grid_core/views/m_rows_view.ts

```diff
@@ -916,7 +916,7 @@ class RowsView extends ColumnsView {
     that._scrollTop = 0;
     that._scrollLeft = -1;
     that._scrollRight = 0;
-    that._hasHeight = false;
+    that._hasHeight = undefined;
```

Comment:
This third value of the flag variable looks like a crutch.
Because as far as I can see we should skip the first flag check.

Maybe we should move the check of this flag and warning log to another place in the ```grid_core``` / ```virtual_scrolling```?

Reply:
Discussed, decided to leave this as is for now